src/component.js

```diff
@@ -136,7 +136,7 @@ const TinyRendererComponentMixin = {
     return this.node;
   },
 
-  receiveComponent() {
+  receiveComponent(nextElement, transaction, context) {
```

Comment:
They're used below in the call to `this.updateChildren`. I was just wondering where those args came from.